### ISSUE TITLE
Stalls after Loading Sources

### ISSUE THREAD

**Issue by rgdawes** [ShammyLevva/FTAnalyzer#33]
I have tried this several times with GEDCOM's created from different programs.  FT Analyzer opens normally and when I open the GEDCOM it loads the sources and then stalls.  This happens on both my Windows 7 Pro 64 desktop and Windows 10 64 laptop.  I have uninstalled FTA and re-installed it without success.  What am I missing?

**Comment by ShammyLevva** [comment]
Sorry to hear you re having problems. Can you confirm what precise version number you are using please. 

It may be there is some form of commonality between the GEDCOMs that causes it to freeze. I'd need to see the GEDCOM file myself to load it whilst in debugging mode to determine what is going on.

**Comment by ShammyLevva** [comment]
Is this still an issue? I cannot fix if I don't have any feedback.

**Comment by rgdawes** [comment]
Yes, I sent a reply to your email which was maybe the wrong way to do this.  Here is my reply
I am using version 6.5.1.2

You can get my GEDCOM from https://drive.google.com/file/d/1VTK5OIeRfkUI0AWoKP6RCG9ymvFzECbb/view?usp=sharing

I have been able to import my GEDCOM into Heredis and Legacy 9 without any problems. 

Please let me know when you have the GEDCOM so I can take down the link.

**Comment by fire-eggs** [comment]
I took the liberty of having a quick look at the GEDCOM.

FTA is getting stuck in an infinite loop trying to parse certain PLAC statements. I'm not certain how they got there, but special characters are causing confusion. In particular, the © and » symbols. I'm not sure if this is a Family Tree Maker 'feature' or a scheme of your own ...

The first example is in the record for I00014. The BURI tag has a PLAC line which reads:
`2 PLAC © 1851, 1861, 1871, 1881, 1891, 1901, 1911`. (line 533)

I stripped all © symbols, then had to strip all » (e.g. line 548: `2 PLAC » Lancing, Sussex, England`). Then FTA would read the file. There are a bunch of ® symbols as well but they don't seem to be the problem.

So if you edit the file to strip those characters, you can work around the problem, pending a fix in FTA. Note though the result will be large numbers of "weird" places: i.e. for I00014 FTA will interpret the burial place as something like "town=1851, county=1861, country=1871".

**Comment by rgdawes** [comment]
Right, now I get it.  I have been using symbols as a source shorthand on my main Family Tree Maker screen so I don't have to constantly open the source information.  I use ® for a vital record certificate, ¿ for other official transcripts, © (+ year) for census returns and » or « for IGI references with or without batch numbers.  Now that I understand why it's hanging, I can easily edit out the symbols in Notepad before running FTA.  Thanks for figuring this out.

**Comment by ShammyLevva** [comment]
Thanks for the heads up.

**Comment by ShammyLevva** [comment]
Apologies it's been a massively busy last few weeks I just tried to access your GEDCOM and it says it's not available.

**Comment by rgdawes** [comment]
Sorry, I thought you were finished with it so I took it down.  Here it is again
https://drive.google.com/file/d/1MG7I2zsEOo9MbmkHHqltzL917g-NkRCb/view?usp=sharing
